playground/Directory.Build.props

```diff
@@ -9,6 +13,8 @@
       CS1712: Type parameter 'type_parameter' has no matching typeparam tag in the XML comment on 'type_or_member' (but other type parameters do)
     -->
     <NoWarn>$(NoWarn),1573,1591,1712</NoWarn>
+
+    <SharedDir Condition="'$(SharedDir)' == ''">$(MSBuildThisFileDirectory)..\src\Shared\</SharedDir>
```

Comment:
Why isn't this already set from the root Directory.Build.props?

Reply:
This is to cover the case when `playground` is being built from outside the repo, and thus doesn't have that root Directory.Build.props .
I'm not super happy with this, and some others we'll need for playground, but I don't overdo that at least just yet!

Reply:
Is it needed for more than KnownResourceNames.cs?

Looks like that only contains a single const string

https://github.com/dotnet/aspire/blob/11e90df77649e3abeab69b37912b6fd7f449f50a/src/Shared/KnownResourceNames.cs#L8

Maybe we just copy that string into a `playground/KnownResourceNames.cs` file. It's just a simple string. Easier to copy it once than to try to copy around files at build time.

Reply:
btw, there will be a bunch of copying around done on CI when archiving the playground tests. But this one is clearly a one-off, so I pushed the change as suggested.